cpu/stm32f1/periph/rtt.c

```diff
@@ -151,7 +151,7 @@ void rtt_clear_alarm(void)
 
 void rtt_poweron(void)
 {
-    periph_clk_en(APB1, (RCC_APB1ENR_BKPEN|RCC_APB1ENR_PWREN)); /* enable BKP and PWR, Clock */
+    periph_clk_en(APB1, (RCC_APB1ENR_BKPEN)); /* enable BKP, Clock */
```

Comment:
Nit-picky and I won't block for that, but you could have removed the parenthesis here and below

Reply:
done